<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LangStyleDefineClass</class>
 <widget class="QMainWindow" name="LangStyleDefineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LangStyleDefine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Setting</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Definition Language</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="curDefineLangCb"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Mother Language</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="motherLangCb">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cpp</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="delBt">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="newBt">
           <property name="text">
            <string>New Create</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="saveBt">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="saveAsBt">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Expand File Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="extNameLe">
           <property name="maxLength">
            <number>256</number>
           </property>
           <property name="placeholderText">
            <string>js cs (split with spaceï¼‰</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Input Key Words</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="keyWordEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>newBt</sender>
   <signal>clicked()</signal>
   <receiver>LangStyleDefineClass</receiver>
   <slot>slot_new()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>547</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBt</sender>
   <signal>clicked()</signal>
   <receiver>LangStyleDefineClass</receiver>
   <slot>slot_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAsBt</sender>
   <signal>clicked()</signal>
   <receiver>LangStyleDefineClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delBt</sender>
   <signal>clicked()</signal>
   <receiver>LangStyleDefineClass</receiver>
   <slot>slot_delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_new()</slot>
  <slot>slot_save()</slot>
  <slot>slot_delete()</slot>
 </slots>
</ui>
